examples/translations.json

```diff
@@ -0,0 +1,107 @@
+{   "_note" : "Filtrer for this listing request:[{\"property\":\"translated_on\",\"operator\":\"exists\",\"value\":false,\"type\":\"boolean\"},{\"property\":\"source\",\"value\":\"article\"},{\"property\":\"source_id\",\"value\":\"054824\"}]",
```

Comment:
This filter should be used to highlight the "translation" icon (so that the user knows that a translation has been already requested), and permits the delete (using the id)

```
[
  {"property": "translated_on", "operator": "exists", "value": false, "type": "boolean"},
  {"property": "source", "value": "article"},
  {"property": "source_id", "value": "054824"}
]
```

This kind of filtering give us the possibility later on to an history of the translations on a specific field